package.json

```diff
@@ -15,6 +15,8 @@
     "start": "start-storybook -p 6006",
     "generate": "concurrently yarn:generate:*",
     "generate:tokens": "esbuild scripts/generate-tokens.ts --bundle --platform=node --target=es2019 --format=cjs --external:theo | node",
+    "generate:theme:day": "esbuild scripts/generate-theme.ts --bundle --platform=node --target=es2019 --format=cjs --external:theo | node",
+    "generate:theme:night": "esbuild scripts/generate-theme.ts --bundle --platform=node --target=es2019 --format=cjs --external:theo | node",
```

Comment:
I do get the theme name from script name, e.g. "day" or "night".

This however then requires having two scripts set, even though their executions are the same.

I could have done only with one script where I keep an array of theme names, however I feel like this is more powerfull e.g. you could run a script for only one theme.

Reply:
Yea I like that, more flexible.